modules/claims_api/app/swagger/claims_api/v2/swagger.json

```diff
@@ -0,0 +1,599 @@
+                                "type": "boolean",
+                                "description": "The claim filer has requested a claim decision be made",
+                                "example": "false"
+                              },
+                              "claim_type": {
+                                "type": "string",
+                                "description": "The type of claim originally submitted",
+                                "enum": [
+                                  "Compensation",
+                                  "Compensation and Pension",
+                                  "Dependency",
+                                  "Administrative Review"
+                                ],
+                                "example": "Compensation"
+                              }
+                            }
+                          }
+                        }
+                      }
+                    }
+                  }
+                }
+              }
+            }
+          },
+          "401": {
+            "description": "Unauthorized",
+            "content": {
+              "application/json": {
+                "example": {
+                  "errors": [
+                    {
+                      "title": "Not authorized",
+                      "detail": "Not authorized",
+                      "code": "401",
+                      "status": "401"
+                    }
+                  ]
+                },
+                "schema": {
+                  "required": [
+                    "errors"
+                  ],
+                  "properties": {
+                    "errors": {
+                      "type": "array",
+                      "items": {
+                        "additionalProperties": false,
+                        "required": [
+                          "title",
+                          "detail",
+                          "code",
+                          "status"
+                        ],
+                        "properties": {
+                          "title": {
+                            "type": "string",
+                            "description": "HTTP error title"
+                          },
+                          "detail": {
+                            "type": "string",
+                            "description": "HTTP error detail"
+                          },
+                          "code": {
+                            "type": "string",
+                            "description": "HTTP error code"
+                          },
+                          "status": {
+                            "type": "string",
+                            "description": "HTTP error code"
+                          }
+                        }
+                      }
+                    }
+                  }
+                }
+              }
+            }
+          },
+          "404": {
```

Comment:
i know this isnt necessarily part of this specific ticket, but can we take this opportunity to remove 404 from the possible responses this claims endpoint can respond with?